parquet-test-utils/src/lib.rs

```diff
@@ -58,6 +58,16 @@ pub struct ParquetScanOptions {
     pub enable_page_index: bool,
 }
 
+impl ParquetScanOptions {
+    /// Returns a [`SessionConfig`] with the given options
+    pub fn config(&self) -> SessionConfig {
+        SessionConfig::new()
```

Comment:
I debated simply removing `ParquetScanOptions` in favour of `SessionConfig` but figured this PR was large enough as it was

Reply:
Thank you. I agree this PR is already large. I also think the ParquetScanOptions predated the config options.

I think removing the ParquetScanOptions as a follow on PR is a good idea 👍